ETC.tsv

```diff
@@ -2551,7 +2551,7 @@ ETC_20150317_002550	{memo X}Obtained Mayvern's key!
 ETC_20150317_002551	{memo X}Naktis' subordinates are being too disrupting!
 ETC_20150317_002552	{memo X}Using a key of Mayvern..
 ETC_20150317_002553	Experimental Mesyum
-ETC_20150317_002554	$Sprinkling the Acidic Solution
+ETC_20150317_002554	$Pouring the Acidic Solution
```

Comment:
I think the meaning of the word doesn't work as well in English, and has a bit broader meaning in Korean:
https://www.google.com/search?q=images&biw=1920&bih=991&source=lnms&tbm=isch&sa=X&ved=0CAYQ_AUoAWoVChMIzYPJvN6zyAIVitGACh1kqgoU#tbm=isch&q=Sprinkling
Notice how it's only out of a container with a cover that has perforated holes, or from a hand that is distributing the object by moving it's fingers back and forth.
https://www.google.com/search?q=images&es_sm=93&biw=1920&bih=991&source=lnms&tbm=isch&sa=X&ved=0CAYQ_AUoAWoVChMIzbv15N6zyAIVyM6ACh0ljArI#tbm=isch&q=%EB%BF%8C%EB%A6%AC%EA%B8%B0
Here, it seems to cover "spread, splatter, pour, splash" all of which you wouldn't ever replace with "sprinkle" in English.